tmk_core/protocol/usb_descriptor.h

```diff
@@ -224,6 +234,10 @@ enum usb_endpoints {
 #    define CDC_IN_EPADDR (ENDPOINT_DIR_IN | CDC_IN_EPNUM)
 #    define CDC_OUT_EPADDR (ENDPOINT_DIR_OUT | CDC_OUT_EPNUM)
 #endif
+#ifdef JOYSTICK_ENABLE
+    JOYSTICK_IN_EPNUM  = NEXT_EPNUM,
+    JOYSTICK_OUT_EPNUM = NEXT_EPNUM,
```

Comment:
```suggestion
```

Reply:
I merged your changes removing the output capacity on the joystick endpoint, but it seems to create a build error : 
tmk_core/protocol/chibios/usb_main.c:297:46: error: 'JOYSTICK_OUT_CAPACITY' undeclared here (not in a function); did you mean 'JOYSTICK_IN_CAPACITY'?
     .joystick_driver = QMK_USB_DRIVER_CONFIG(JOYSTICK, 0, false),
                                              ^~~~~~~~
tmk_core/protocol/chibios/usb_main.c:184:74: note: in definition of macro 'QMK_USB_DRIVER_CONFIG'
         .queue_capacity_in = stream##_IN_CAPACITY, .queue_capacity_out = stream##_OUT_CAPACITY,

It seems the output capacity is expected by the macro.
Either I don't use the macro to define the endpoint and do it manually (or with a macro that defines only the input capacity), if that's possible, or I revert your changes to keep the output capacity although it won't be used. Which one to you prefer?

Reply:
Ah yeah...oops. The out mode and capacity should go back as they were (even though there is only device-to-host communication for the joystick endpoint).